spec/system/user_logs_in_with_password_spec.rb

```diff
@@ -27,20 +27,6 @@ def submit_login_form(email, password)
       expect(page).to have_text("Invalid Email or password.")
     end
 
-    it "displays a message on the last login attempt" do
```

Comment:
These two specs got removed because the whole point of Devise's paranoid mode is not exposing information like this.